### PR TITLE
Chainlink

### PR DESCRIPTION
**Pull request by nottrunner**
Chainlink Governance Research Draft

### CODE REVIEW

File: Chainlink [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+
+The Node Operators will be able to receive LINK tokens as form of payment for providing a 24/7 reliable source for the API feeds they choose to offer, and also the option to “stake” their LINK tokens for [collatera](https://github.com/smartcontractkit/chainlink/wiki/Protocol-Information#penalty-deposits)l, which will be requested in some service level agreements.
+
+The Smart Contracts Developers will have end-to-end reliable contracts by confirming the data feed was pulled from their desired source(s). The API Providers will benefit from a new lucrative way to sell their data.
+
+Chainlink nodes are not the conventional “stake and forget” nodes that we are used to from various Proof of Stake (“PoS”) coins, which require little to no participation once you have staked them. Chainlink nodes need to offer information which is provided via an API (Application Programming Interface).
```
Comment (RichardRed0x) [inline Chainlink]: This text appears in [Altcoin Magazine](https://medium.com/altcoin-magazine/earning-your-way-in-the-trustless-future-chainlink-ecosystem-101-2922d56c9bfa), so should be a quote block with a link as reference.

File: Chainlink [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+
+"This means that besides having the node running, you also need to know what you are aiming to offer with your node — which API source you are going to offer data from. There may be some APIs that are offered for free, which can be in demand, but competition from other nodes will be higher due to the low barrier to entry. It is likely that most API sources that will have demand will be the ones you need to pay for — usually annual/monthly purchase of the API keys. Keep in mind that re-selling data via a third party API without an agreement is generally against the Terms of Service of the API, and may result in the API provider cutting your access to the API, which can lead to penalty payments for your node." [https://medium.com/altcoin-magazine/earning-your-way-in-the-trustless-future-chainlink-ecosystem-101-2922d56c9bfa](url)
```
Comment (RichardRed0x) [inline Chainlink]: Quote blocks (">") please.

File: Chainlink [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+
+The Chainlink's testnet has been live since [May 18 2014](https://medium.com/coinmonks/technical-overview-of-the-chainlink-testnet-implementation-c8a0b09276d1). Mainnet will be launched on [May 30 2019](https://www.youtube.com/watch?v=oBjq-cV2BkI).
```
Comment (RichardRed0x) [inline Chainlink]: Has it launched now? A good link to the mainnet would add value here.

File: Chainlink [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+
+Chainlink will govern the network through a process it calls validation. The ChainLink Validation System monitors on-chain oracle behavior, providing an objective performance metric that can guide user selection of oracles. It will seek to monitor oracles for:
+
+- Availability: The Validation System should record failures by an oracle to respond in a timely way to queries. It will compile ongoing uptime statistics.
+
+- Correctness: The Validation System should record apparent erroneous responses by an oracle as measured by deviations from responses provided by peers
+
+In the initial, on-chain aggregation system in ChainLink, such monitoring is straightforward, as all oracle activity is visible to CHAINLINK. There is also an off-chain aggregation system envisaged for ChainLink where the oracles themselves perform aggregation. 
+
+Consequently, Chainlink not have direct visibility into oracle responses and cannot itself monitor avail-ability and correctness. Fortunately, oracles digitally sign their responses, and thus, as a side effect, generate non-repudiable evidence of their answers.
```
Comment (RichardRed0x) [inline Chainlink]: > Chainlink not have

File: Chainlink [hunk 5]
```diff
@@ -0,0 +1,236 @@
...
+
+Chainlink’s proposed approach would be to realize the validation service as a smart contract that would reward oracles for submitting evidence of deviating responses.  In other words, oracles would be incentivized to report apparently erroneous behavior. 
+
+Availability is somewhat trickier to monitor, as oracles of course don’t sign their failures to respond. Instead, a proposed protocol enhancement would require oracles to digitally sign attestations to the set of responses they have received from other oracles. 
+
+The validation contract would then accept (and again reward) submission of sets of attestations that demonstrate consistent non-responsiveness by an under per-forming oracle to its peers. In both the on-chain and off-chain cases, availability and correctness statistics of oracles will be visible on-chain. Users / developers will thus be able to view them in real time through an appropriate front end, such as a Dapp in Ethereum or an equivalent application for a permissioned blockchain.
```
Comment (RichardRed0x) [inline Chainlink]: > per-forming
There are a few of these, could be worth pasting from source again.

File: Chainlink [hunk 6]
```diff
@@ -0,0 +1,236 @@
...
+
+**How are funds managed?**
+
+Funds are managed by the Chainlink team.
+
+**How is work other than software development funded?**
+
+Work other than software development is also funded by the development fund managed by Chainlink 
+ 
+**Related projects**
```
Comment (RichardRed0x) [inline Chainlink]: I would ditch these unless they are related directly to Chainlink - i.e. not just similar

File: Chainlink (hunk 2 above)
Comment (nottrunner) [reply Chainlink]: Updated

File: Chainlink (hunk 3 above)
Comment (nottrunner) [reply Chainlink]: Added a link to mainnet docs and removed reference to testnet

File: Chainlink (hunk 4 above)
Comment (nottrunner) [reply Chainlink]: Added "does" not

File: Chainlink (hunk 6 above)
Comment (nottrunner) [reply Chainlink]: These are projects directly integrating with Chainlink.  They are not competitors.

**Pull request merged**